<launch>
    <node pkg="rr_common" type="mapper" name="mapper" output="screen">
        <param name="sources" value="/scan/pointcloud"/>
        <param name="destination" value="/current_obstacles"/>
        <param name="refresh_rate" value="15"/>
        <param name="combined_frame" value="base_footprint"/>

        <param name="ground_threshold" value="0"/>
        <param name="outlier_neighbors" value="1"/>
        <param name="outlier_threshold" value="100.0"/>
        <param name="vg_filter_size" value="0.05"/> <!-- used 0.1 at AVC 2018 -->
    </node>
</launch>
